### PR TITLE
[SPARK-32850][TEST][FOLLOWUP] Fix flaky test due to timeout

### PR DESCRIPTION
**Pull request by ulysses-you**
### What changes were proposed in this pull request?

Increase test timeout.

### Why are the changes needed?

It's more reasonable to use 60s instead of 6s since many code place use the 60s in `TestUtils.waitUntilExecutorsUp`.

### Does this PR introduce _any_ user-facing change?

No.

### How was this patch tested?

Pass exists test.

### CODE REVIEW

**Comment by ulysses-you** [comment]
cc @HyukjinKwon @dongjoon-hyun @Ngone51

File: core/src/test/scala/org/apache/spark/storage/BlockManagerDecommissionIntegrationSuite.scala [hunk 1]
```diff
@@ -49,7 +49,7 @@ class BlockManagerDecommissionIntegrationSuite extends SparkFunSuite with LocalS
         .set(config.DECOMMISSION_ENABLED, true)
         .set(config.STORAGE_DECOMMISSION_ENABLED, isEnabled)
       sc = new SparkContext(conf)
-      TestUtils.waitUntilExecutorsUp(sc, 2, 6000)
+      TestUtils.waitUntilExecutorsUp(sc, 2, 60000)
```
Comment (Ngone51) [inline core/src/test/scala/org/apache/spark/storage/BlockManagerDecommissionIntegrationSuite.scala:52]: Oops..I wondered why it's flaky at this place before and I thought it was already 60s....

**Review by Ngone51 — APPROVED** [review]
LGTM! Thanks for the fix!

**Review by HyukjinKwon — APPROVED** [review]

File: core/src/test/scala/org/apache/spark/storage/BlockManagerDecommissionIntegrationSuite.scala (hunk 1 above)
Comment (ulysses-you) [reply core/src/test/scala/org/apache/spark/storage/BlockManagerDecommissionIntegrationSuite.scala:52]: ah, I guess it's a typo : )

**Comment by HyukjinKwon** [comment]
Merged to master and branch-3.1.

**Comment by ulysses-you** [comment]
thanks for merging !

**Pull request closed without merging**